---
layout: default
title: About Me
---

		<div class="about">
			<h1>About Me</h1>
			<p>My main interests involve the genetic basis of adaptation and speciation. I predominantly use genomic methods to understand these processes but have also produced new genomic resources for non-model study systems, including chromosome-scale reference genomes, and have carried out microbiome studies to understand how microbes may facilitate adaptation and speciation.</p>
			<p>I am currently a postdoc at the University of Edinburgh funded by an Early Postdoc Mobility fellowship from the Swiss Science Foundation (SNSF) working on understanding the evolution and maintanence of a wing-pattern supergene in the African Monarch butterfly <i>Danaus chrysippus</i>.</p>
			<p>Prior to my postdoc I completed my PhD at Eawag & the University of Bern in Switzerland under the supervision of Philine Feulner and Ole Seehausen and my research interests are mainly related to the genomic patterns underpinning adaptation and speciation. My PhD project is focussed on understanding genomic divergence between members of the diverse Swiss whitefish radiation (<em>Coregonus lavaretus</em> species complex). Specifically I hope to identify differences patterns of divergence, trait architecture and genome structure between ecologically different sympatric whitefish species within the radiation and compare and contrast these patterns with the differences between ecologically similar allopatric species from different lake systems.</p>
			<p>The first part of my project involved the production of a high resolution linkage map for Alpine whitefish using F1 offspring from two wild-caught <em>C. sp. "Albock"</em> individuals from lake Thun. The map was produced using a RAD-seq approach and the linkage mapping software Lep-MAP3 resulting in both sex-specific and sex-averaged maps containing ~6000 SNP loci. (the resulting paper can be found <a href="https://www.g3journal.org/content/ggg/8/12/3745.full.pdf">HERE</a>)</p>
			<p style="text-align:center;"><img src="/pics/LM.png" width="800" height="450" class="center">
			<p>For the second chapter of my project I assembled the Alpine whitefish genome, which, being large and still residually tetraploid, posed many challenges. We carried out ~90x PacBio sequencing and then used three assemblers: Falcon, Canu and wtdbg2. After Hi-C scaffolding, each of the assembles was of high quality and each included >90% BUSCOs. The best assembly was determined to be that from wtdbg2 which I then went on to annotate using MAKER2. (Our Alpine whitefish genome paper can be found <a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.13187">HERE</a>)</p>
			<p style="text-align:center;"><img src="/pics/Figure5.png" width="600" height="600" class="center">
			<p>The final part of my project involved the analysis of a whole-genome resequencing dataset which included 97 Alpine whitefish individuals from 21 species from across the radiation. We have sequenced three individuals of each of the species from each lake with a radiation to around 20x coverage. I now aim to understand how these whitefish species are related to one another, the extent of gene flow across the radiation and determine the genetic basis and possible parallelism in key ecological traits.</p> 
			<ul class="contacts">
				</ul>
		</div>
