---
layout: default
title: About Me
---
<p style="background-image: url('/pics/palm.jpg');">
		<div class="about">
			<h1>About Me</h1>
			<p>Hi, I'm Rishi - a 27 year old postdoc from the UK. 
			<p><u><strong>My main interests involve:</strong></u> understanding variation in genome structure and the genetic basis of key traits, adaptation, and speciation -  which I have predominantly investigated using genomic methods. </p>
			<p><u><strong>I have investigated these questions in:</strong></u> <i>Howea</i> palm trees (and their associated soil microbial communities), sharks, Alpine whitefish, and most recently, <i>Danaus</i> butterflies. </p>
			<p><u><strong>I've been lucky enough to carry out this work at:</strong></u> the National Botanic Garden of Wales, Imperial College London, Kew Gardens, Eawag (the Swiss federal department for aquatic research), and currently the University of Edinburgh.</p>
			<p>I am currently funded by an Early Postdoc Mobility fellowship from the Swiss Science Foundation (SNSF) and my project aims to understand the evolution and maintanence of a wing-pattern supergene in the African Monarch butterfly <i>Danaus chrysippus</i>. This work spans a number of topics including the annotation of multiple <i>Danaus chrysippus</i> reference genome assemblies, and using these assemblies and annotations to understand how different supergene alleles (that result in different wing-patterns) have evolved.</p>
			<p style="text-align:center;"><img src="/pics/martin_lab_pic.jpeg" width="600" height="450" class="center">
			<br>
			<p>Prior to my postdoc I completed my PhD at Eawag & the University of Bern in Switzerland under the supervision of Philine Feulner and Ole Seehausen and my research interests are mainly related to the genomic patterns underpinning adaptation and speciation. My PhD project was focussed on understanding genomic divergence between members of the diverse Alpine whitefish radiation (<em>Coregonus lavaretus</em> species complex). </p>
			<p>The first part of my project involved the production of a high resolution linkage map for Alpine whitefish using F1 offspring from two wild-caught <em>C. sp. "Albock"</em> individuals from lake Thun. The map was produced using a RAD-seq approach and the linkage mapping software Lep-MAP3 resulting in both sex-specific and sex-averaged maps containing ~6000 SNP loci. (the resulting paper can be found <a href="https://www.g3journal.org/content/ggg/8/12/3745.full.pdf">HERE</a>)</p>
			<p style="text-align:center;"><img src="/pics/LM.png" width="800" height="450" class="center">
			<p>For the second chapter of my project I assembled the Alpine whitefish genome, which, being large and still residually tetraploid, posed many challenges. We carried out ~90x PacBio sequencing and then used three assemblers: Falcon, Canu and wtdbg2. After Hi-C scaffolding, each of the assembles was of high quality and each included >90% BUSCOs. The best assembly was determined to be that from wtdbg2 which I then went on to annotate using MAKER2. (Our Alpine whitefish genome paper can be found <a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.13187">HERE</a>)</p>
			<p style="text-align:center;"><img src="/pics/Figure5.png" width="600" height="600" class="center">
			<p>The final part of my project involved the analysis of a whole-genome resequencing dataset which included 97 Alpine whitefish individuals from 21 species from across the radiation. We have sequenced three individuals of each of the species from each lake with a radiation to around 20x coverage. I now aim to understand how these whitefish species are related to one another, the extent of gene flow across the radiation and determine the genetic basis and possible parallelism in key ecological traits.</p> 
			<ul class="contacts">
				</ul>
		</div>
